kovri.md

```diff
@@ -1,64 +1,68 @@
-Kovri Lightweight C++ I2P Router Script
-
-Version 1 - sgp
-Version 2 - scoobs
-Version 3 - sgp
+The Kovri Script
 
 Until this point, this video series has covered how Monero obfuscates information stored on the blockchain.
 Ring signatures obscure the sender.
-Stealth addressing prevents anyone but the sender and the recipient from knowing the recipient owns Monero.
-RingCT hides the amount of Monero transmitted.
-However, some personally identifiable information can still be leaked in the process of sending Monero.
-This video will focus on a technology that closes that gap, called Kovri.
+Stealth addressing prevents anyone, other than the sender and the recipient, from knowing that the recipient owns Monero.
+RingCT hides the amount of Monero that's transmitted.
+However, some personally identifiable information will still be leaked in the process of sending Monero.
+This video focuses on the technology that closes this gap of privacy. This technology is called Kovri.
```

Comment:
I think I prefer this as a single sentence.

Reply:
>This video will focus on a technology that closes that gap, called Kovri.

Prior to the revision, even with the comma, this sentence too closely implied that Kovri was that gap.

Let's try a 3rd revision.

Reply:
I see your point. Good catch.